Update linting scripts in package.json removed the redundant code

MohSay890 opened the pull request:
What I changed and why:

1. Created `lint:js`: I moved the specific JavaScript check to its own command `lint:js`.

2. Updated `lint`: The main lint command now chains them together `lint:js && lint:html`. This means when the GitHub Action runs `npm run lint`, it will now check both.

3. Updated `lint:all`: I just pointed this to `npm run lint` so it does the same thing, keeping it as an alias in case anyone is used to typing it.

4. Remove `express` from `dependencies` since it is already in `devDependencies`

MohSay890 committed:
Update linting scripts in package.json removed the redundant code

TheBestPartOfMe commented on package.json:
```diff
@@ -3,18 +3,19 @@
-    "lint": "eslint '*.js'",
+    "lint": "npm run lint:js && npm run lint:html",
+    "lint:js": "eslint '*.js'",
     "lint:fix": "eslint '*.js' --fix",
```
```suggestion
    "lint:js-fix": "eslint '*.js' --fix",
```
Vielleicht könnten wir durch diese Änderung klarmachen, dass es der fix für den JS Linter ist

MohSay890 committed:
Update package.json

Guter Punkt. Die Umbenennung in lint:js-fix sorgt für Konsistenz mit lint:js. Done :-)

Co-authored-by: Max <48136746+TheBestPartOfMe@users.noreply.github.com>

TheBestPartOfMe commented on package.json:
```diff
@@ -3,18 +3,19 @@
     "test": "jest",
-    "lint": "eslint '*.js'",
-    "lint:fix": "eslint '*.js' --fix",
+    "lint": "npm run lint:js && npm run lint:html",
+    "lint:js": "eslint '*.js'",
+    "lint:js-fix": "eslint '*.js' --fix",
```
```suggestion
    "lint:": "eslint '*.js'",
    "lint:js-fix": "eslint '*.js' --fix",
```

TheBestPartOfMe reviewed (changes requested):
Da @ismne21 ja bereit eine extra Action angelegt hat, damit man granularer unterscheiden kann, welcher Lint erfolgreich war, würde ich diesen PR nur noch für das Richtigstellen der dependencies nutzen wollen. Die Vorschläge sollen den "alten" Zustand wiederherstellen, also `lint` für js lint und `lint:html` für den html lint.

TheBestPartOfMe commented on package.json:
```diff
@@ -3,18 +3,19 @@
+    "lint:js-fix": "eslint '*.js' --fix",
     "lint:html": "htmlhint index.html",
-    "lint:all": "npm run lint && npm run lint:html"
+    "lint:all": "npm run lint"
```
```suggestion
   "lint:all": "npm run lint && npm run lint:html"
```
